src/sql/azdata.d.ts

```diff
@@ -756,9 +756,7 @@ declare module 'azdata' {
 		flavor: string;
 	}
 
-	export interface QueryExecutionOptions {
-		options: Map<string, any>;
-	}
+	export interface QueryExecutionOptions { [option: string]: any; }
```

Comment:
i think you mean for this to be
```
{
      options: {[option: string]: any;}
}
````

Reply:
Thanks @anthonydresser! Would've definitely not caught that without what you said.